mantle/cmd/kola/testiso.go

```diff
@@ -416,6 +522,13 @@ func awaitCompletion(inst *platform.QemuInstance, outdir string, qchan *os.File,
 				errchan <- fmt.Errorf("Unexpected string from completion channel: %s expected: %s", line, exp)
 				return
 			}
+			// switch the boot order here, we are well into the installation process. Only used for PXE test now
+			if line == liveOKSignal {
```

Comment:
This is completely fine for now though I think the functionality would better live in `metal.go` - that's intending to be the abstraction over performing an install, whereas `testiso.go` is just about testing.

Concretely eventually we want `kola run -p pxe-install podman.*` and `cosa run -p pxe-install` which would run tests/give you ssh after doing a PXE install, rather than how `testiso.go` is hardcoded to just a sanity check.

Fine for now though.

Reply:
i agree. I was going to do some refactoring around this and try to move everything to metal. But - for this particular case, is there a better place in `metal.go` to figure out that we have started with the installation process and can switch the boot order?

Reply:
It's totally fine as is, feel free to just add a comment for this and we can do it later.  Given the size of change here I'd prefer to get it in than force you into rebasing later etc.

But if you want to take a quick shot at it I think what `metal.go` should do is inject its own Ignition to write to a separate virtio channel, distinct from what `testiso.go` is doing.

Reply:
ah..ok got it ..yes..i'll try to tackle that separately :) i'll stick with these changes for now and probably move some of the QMP bits to its own package.